### PR TITLE
Adds prow job config

### PR DESCRIPTION
**Pull request by simonswine**

### CODE REVIEW

**Comment by simonswine** [comment]
@kragniz if you take a quick look, if that makes sense. 1.8 is expected to fail 🙂

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+BUNDLE_DIR ?= .bundle
+
+verify: bundle_install
+	bundle exec rake test
+
+bundle_install:
+	bundle install --path $(BUNDLE_DIR)
+
+acecptance: acceptance-1-7-centos
```
Comment (kragniz) [inline Makefile]: typo on `acecptance`

**Review by kragniz — CHANGES_REQUESTED** [review]
typo, otherwise looks good

File: Makefile (hunk 1 above)
Comment (simonswine) [reply Makefile]: Good spot!

**Pull request merged**